lib/Cabal2Nix.hs

```diff
@@ -243,16 +257,18 @@ instance ToNixExpr CompilerFlavor where
   toNix flavour = mkSym "compiler" @. (fromString . ("is" ++) . capitalize . show . pretty $ flavour)
 
 instance ToNixExpr VersionRange where
```

Comment:
I think this should stay. And `projectVersionRange` should be added to the callsite. Or an `instance ToNixExpr` added for what ever `projectVersionRange`s datatype is.

Reply:
Done